src/routes.js

```diff
@@ -1417,6 +1420,101 @@ module.exports = async (
     sentPayment.write(paymentRequest);
   });
 
+  app.post("/api/lnd/trackpayment", async (req, res) => {
+    const { router } = LightningServices.services;
+    const { paymentHash, inflightUpdates = true } = req.body;
+    if (req.limituser) {
+      const health = await checkHealth();
+      if (health.LNDStatus.success) {
+        res.sendStatus(403);
+      } else {
+        res.status(500);
+        res.json({ errorMessage: "LND is down" });
+      }
+    }
+
+    logger.info("Tracking payment payment", { paymentHash, inflightUpdates });
+    const trackedPayment = router.trackPaymentV2({
+      payment_hash: paymentHash,
+      no_inflight_updates: !inflightUpdates
+    });
+
+    trackedPayment.on("data", response => {
+      if (response.payment_error) {
+        logger.error("TrackPayment Info:", response)
+        return res.status(500).json({
+          errorMessage: response.payment_error
+        });
+      }
+      
+      logger.info("TrackPayment Data:", response);
+      return res.json(response);
+    });
+
+    trackedPayment.on("status", status => {
+      logger.info("TrackPayment Status:", status);
+    });
+
+    trackedPayment.on("error", async err => {
+      logger.error("TrackPayment Error:", err);
+      const health = await checkHealth();
+      if (health.LNDStatus.success) {
+        res.status(500).json({
+          errorMessage: sanitizeLNDError(err.message)
+        });
+      } else {
+        res.status(500);
+        res.json({ errorMessage: "LND is down" });
+      }
+    });
+  });
+
+  app.post("/api/lnd/sendtoroute", async (req, res) => {
+    const { router } = LightningServices.services;
+    const { paymentHash, route } = req.body;
+    if (req.limituser) {
+      const health = await checkHealth();
+      if (health.LNDStatus.success) {
+        res.sendStatus(403);
+      } else {
+        res.status(500);
+        res.json({ errorMessage: "LND is down" });
+      }
+    }
+
+    router.sendToRoute({ payment_hash: paymentHash, route }, (err, data) => {
+      if (err) {
+        logger.error("SendToRoute Error:", err);
+        return res.status(400).json(err);
+      }
+
+      return res.status(200).json(data);
+    });
+  });
+
+  app.post("/api/lnd/estimateroutefee", async (req, res) => {
```

Comment:
This should be a GET

Reply:
? client is posting destination/amt

Reply:
^

Reply:
You are GETting, an estimate route fee FROM the server.

Wether the GET requires input or not is irrelevant.

Only rule is GET(arg1, arg2, ...) should always return the same result (in short 0GETs should not result in database writes).

Reply:
Wrapper is keeping with lnd, get doesn't make sense as the response is dynamic